packages/rendering/src/utils/exifDataAndImageHandler.ts

```diff
@@ -0,0 +1,65 @@
+import path from 'path'
+import { readFileSync, writeFileSync, rm } from 'fs'
+import * as exif from 'piexif-ts'
+
+export function createFileWithCleanExif(sourceFilePath: string): string {
+  const imageData = function (src: string) {
+    return readFileSync(src).toString('binary')
+  }
+  if (['.jpg'].includes(path.extname(sourceFilePath))) {
+    const newFileSource: string = sourceFilePath
+    const data = exif.load(imageData(sourceFilePath))
+    const cleanExifData: exif.IExif = createCleanExif(data)
+    const newImageBinary = exif.insert(exif.dump(cleanExifData), exif.remove(imageData(sourceFilePath)))
+    rm(sourceFilePath, { force: true }, (err) => {
```

Comment:
käytä `fs/promises` niin toimii await `rm(...)`, `await readFile` ja `await writeFile`.

Tuossa on nimittäin semmonen bugi tällä hetkellä, että rm jää ajamaan taustalle vaikka failaisi